runtime/shared_common/CacheMap.cpp

```diff
@@ -288,7 +288,7 @@ SH_CacheMap::initialize(J9JavaVM* vm, J9SharedClassConfig* sharedClassConfig, Bl
 
 	_managers = SH_Managers::newInstance(vm, (SH_Managers *)allocPtr);
 
-	_ccHead = _cc = SH_CompositeCacheImpl::newInstance(vm, sharedClassConfig, (SH_CompositeCacheImpl*)(allocPtr += SH_Managers::getRequiredConstrBytes()), cacheName, cacheTypeRequired, startupForStats);
+	_ccHead = _cc = SH_CompositeCacheImpl::newInstance(vm, sharedClassConfig, (SH_CompositeCacheImpl*)(allocPtr += SH_Managers::getRequiredConstrBytes()), cacheName, cacheTypeRequired, startupForStats, 0);
```

Comment:
Instead of hard coding `0` in this PR, which I expect will need to change later, suggest using a #define, like LAYER_CURRENT, to make them easier to find later.

Reply:
New macro OSCACHE_CURRENT_LAYER_LAYER added